### PR TITLE
19 health fat enhancements stable

### PR DESCRIPTION
**Pull request by ajm01**
latest major drop for microprofile health FAT against stabilization branch

### CODE REVIEW

**Comment by ajm01** [comment]
#build

**Comment by ajm01** [comment]
amy - oh yes, i meant to remove those. I will do that now.

**Comment by scottkurz** [comment]
#build

**Comment by scottkurz** [comment]
PB done against wrong branch.  Doing new PB:
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_1k440J7WEeew8ISV5CV8GQ

**Comment by ajm01** [comment]
fixed issues revealed by earlier PBs - manually submitted a new PB:
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_KQPJ4J8AEeew8ISV5CV8GQ&tab=com.ibm.team.build.web.ui.internal.editors.result.activity.ActivitiesProvider

File: dev/com.ibm.ws.microprofile.health_fat/build.image/wlp/usr/clients/fatTestCommon.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!--
```
Comment (scottkurz) [inline dev/com.ibm.ws.microprofile.health_fat/build.image/wlp/usr/clients/fatTestCommon.xml]: Remove all the `.../build.image/...` stuff, right?

**Comment by ajm01** [comment]
manual PB submitted:
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_vKdfUp8TEeew8ISV5CV8GQ&tab=com.ibm.team.build.web.ui.internal.editors.result.overview.OverviewPage

**Comment by scottkurz** [comment]
Fixes issue #19

**Pull request merged**